examples/text_labels_and_annotations/arrow_demo.py

```diff
@@ -42,12 +42,12 @@ def make_arrow_graph(ax, data, size=4, display='length', shape='right',
         `.FancyArrow` properties, e.g. *linewidth* or *edgecolor*.
     """
 
-    ax.set(xlim=(-0.5, 1.5), ylim=(-0.5, 1.5), xticks=[], yticks=[])
+    ax.set(xlim=(-0.25, 1.25), ylim=(-0.25, 1.25), xticks=[], yticks=[])
     ax.text(.01, .01, f'flux encoded as arrow {display}',
```

Comment:
The description looks even more out of place now that there's less distance to the arrows. I suggest to change this into a title.

Reply:
Sure, that works too.